pavement.py

```diff
@@ -749,4 +831,5 @@ def justcopy(origin, target):
     elif os.path.isfile(origin):
         if not os.path.exists(target):
             os.makedirs(target)
-        shutil.copy(origin, target)
+
+        shutil.copy(origin, target)
```

Comment:
@lucernae was talking about one new line after that one, not before ;-) Always one empty line at the end of a python file

Reply:
oops...
thanks for the feedback @Gustry
wondering why it could pass the travis before.

Reply:
I don't think travis is checking code layout on this `pavement.py`, it's only a deployment file. I would say only on the django app itself. (I don't know, I've never checked the travis config)

Reply:
Yes, travis didn't do pep8 check for geonode. But we should still do the best practice :+1: 
GeoNode upstream also gradually try to follow pep8 guideline but haven't put the test yet on travis.

Reply:
nice explanations.
thanks @lucernae @Gustry